sdk/digitaltwins/Azure.DigitalTwins.Core/src/Serialization/ModelProperties.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Azure.DigitalTwins.Core.Serialization
+{
+    /// <summary>
+    /// Properties on a digital twin that adhere to a specific model.
+    /// </summary>
+    public class ModelProperties
```

Comment:
I realized I had modeled BasicDigitalTwin wrong, or it changed in the last update on me.

Either way, what I found is that both the base part of the digitaltwin type has these 2 properties, as does the properties for a component. So I pulled it out so it can be used in both places.